pkg/trie/triedb/node_storage.go

```diff
@@ -9,6 +9,9 @@ import (
 	"github.com/gammazero/deque"
 )
 
+var emptyNode = make([]byte, 1)
+var hashedNullNode = common.MustBlake2bHash(emptyNode)
```

Comment:
You can define it directly
```suggestion
var emptyNode = []byte{0}
var hashedNullNode = common.MustBlake2bHash(emptyNode)
```